tests/diff_test/BUILD

```diff
@@ -14,6 +14,7 @@ sh_test(
         "//tests:unittest.bash",
     ],
     deps = ["@bazel_tools//tools/bash/runfiles"],
+    tags = ["local"],
```

Comment:
Do you want to comment on why this is needed?

Reply:
Because the test uses bazel, which is otherwise not a test dependency of the test.

I've added a comment.